providers/collection_package.rb

```diff
@@ -189,72 +280,92 @@ def create_collection_package
-		sb_ifalias_el = package_el.add_element('storeByIfAlias')
-		sb_ifalias_el.add_text('true')
-	end
-	if new_resource.store_by_node_id != 'normal'
-		sb_nodeid_el = package_el.add_element('storeByNodeID')
-		sb_nodeid_el.add_text(new_resource.store_by_node_id)
-	end
-	unless new_resource.if_alias_domain.nil?
-		ifalias_domain_el = package_el.add_element('ifAliasDomain')
-		ifalias_domain_el.add_text(new_resource.if_alias_domain)
-	end
-	if new_resource.stor_flag_override == true
-		stor_flag_override_el = package_el.add_element('storFlagOverride')
-		stor_flag_override_el.add_text('true')
-	end
-	unless new_resource.if_alias_comment.nil?
-		ifalias_comment_el = package_el.add_element('ifAliasComment')
-		ifalias_comment_el.add_text(new_resource.if_alias_comment)
-	end
-	unless new_resource.outage_calendars.nil?
-		new_resource.outage_calendars.each do |oc|
-			if package_el.elements["outage-calendar[text() = '#{oc}']"].nil?
-				oc_el = package_el.add_element ('outage-calendar')
-				oc_el.add_text(oc)
+		unless new_resource.include_urls.nil?
+			new_resource.include_urls.each do |include_url|
+				if package_el.elements["include-url[text() = '#{new_resource.include_urls}']"].nil?
+					include_url_el = package_el.add_element 'include-url'
+					include_url_el.add_text(include_url)
+				end
+			end
+		end
+		unless new_resource.store_by_if_alias.nil?
+			if package_el.elements["storeByIfAlias[text() = '#{new_resource.store_by_if_alias}']"].nil?
+				if new_resource.store_by_if_alias == true
+					sb_ifalias_el = package_el.add_element 'storeByIfAlias'
+					sb_ifalias_el.add_text('true')
+				end
+			end
+		end
+		unless new_resource.store_by_node_id.nil?
+			if package_el.elements["storeByNodeID[text() = '#{new_resource.store_by_node_id}']"].nil?
+				if new_resource.store_by_node_id != 'normal'
+					sb_nodeid_el = package_el.add_element 'storeByNodeID'
+					sb_nodeid_el.add_text(new_resource.store_by_node_id)
+				end
+			end
+		end
+		unless new_resource.if_alias_domain.nil?
+			if package_el.elements["ifAliasDomain[text() = '#{new_resource.if_alias_domain}']"].nil?
+				dm_ifalias_el = package_el.add_element 'ifAliasDomain'
+				dm_ifalias_el.add_text(new_resource.if_alias_domain)
+			end
+		end
+		unless new_resource.stor_flag_override.nil?
+			if package_el.elements["storFlagOverride[text() = '#{new_resource.stor_flag_override}']"].nil?
+				stor_flag_override_el = package_el.add_element 'storFlagOverride'
+				stor_flag_override_el.add_text(new_resource.stor_flag_override)
+			end
+		end
+		unless new_resource.if_alias_comment.nil?
+			if package_el.elements["storFlagOverride[text() = '#{new_resource.stor_flag_override}']"].nil?
+				ifalias_comment_el = package_el.add_element 'ifAliasComment'
+				ifalias_comment_el.add_text(new_resource.if_alias_comment)
+			end
+		end
+		unless new_resource.outage_calendars.nil?
+			new_resource.outage_calendars.each do |oc|
+				if package_el.elements["outage-calendar[text() = '#{oc}']"].nil?
```

Comment:
make sure you insert this element in the proper element order (although I believe this one will work since its supposed to be the last element)